test/integration/integration_test.go

```diff
@@ -1404,7 +1404,7 @@ func (ch *TestClickHouse) testBackupSpecifiedPartition(r *require.Assertions) er
 
 	// Reset the result.
 	result = nil
-	rows, err = ch.chbackend.GetConn().Queryx("SELECT count(0) as count from default.t1 where dt != '2022-01-01")
+	rows, err = ch.chbackend.Select("SELECT count(0) as count from default.t1 where dt != '2022-01-01")
```

Comment:
Why you don't follow my review comments `as is`?
Why you didn't try to look to `integration_tests.go` and not figure out how ch.chbackend.Select works?
Why do you still not run test locally before commit to CI/CD?